EVALUATION.md

```diff
@@ -0,0 +1,65 @@
+The evaluation process is done through the evaluate function that is located in the evaluation.py script (https://github.com/lovodkin93/ucca/blob/master/ucca/evaluation.py).
+A wrapping script of the evaluation.py script is evaluate_standard.py (https://github.com/lovodkin93/ucca/blob/master/scripts/evaluate_standard.py). For more details on how the scripts receives its argument, please write evaluate_standard --help in the prompt.
+The evaluation process compares the gold-standard annotation of a specific passage, with the calculated annotation of that same passage, which is calculated by our model.
+Both passages are of Passage object type, which is an object that contains the connected graph that represents the annotation of the passage.
+Our evaluation includes the recall, precision and F1 scores. The calculation of these scores is done by comparing each edge's labels and yield, which are the literals that are under the edge's child node (if we look at the annotation as a tree).
+We can also do an unlabled evaluation, and then for each edge only its yield will be compared. It is important to know that when there is a remote edge, it is ignored in the yield comparison, but we do look at it when comparing lables of edges.
+Also, when there is an implicit node, edges going into them are evaluated by their parent's yield.
+
+Now let us look more closely at the evaluate function:
+
+The evaluate function receives the following input parameters:
+1. guessed: Passage object to evaluate
+2. ref: reference (gold standard) Passage object to compare to
+3. converter: optional function to apply to passages before evaluation. One can choose to convert to the following formats:
```

Comment:
This converter is actually to convert _from_ these formats, not _to_ them, in case the input files are given in these formats instead of standard XML.